appintro/src/main/res/drawable/appintro_indicator_dot_selected.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:bottom="-0.5dp"
+        android:right="4.5dp"
+        android:left="5dp">
+        <shape android:shape="oval">
+            <solid android:color="@color/appintro_default_unselected_color" />
+            <size
+                android:width="@dimen/appIntroIndicatorSize"
+                android:height="@dimen/appIntroIndicatorSize" />
+        </shape>
+    </item>
+    <item
+        android:right="@dimen/appIntroIndicatorInset"

```

Comment:
We're doubling the space between the dots here. We should fix this as it will create problems on intro with several pages.

Reply:
We can easily change this value in the dimensions file. I have decreased the default inset, so we should be fine